cfMedipsQc.wdl

```diff
@@ -400,12 +402,22 @@ task extractMedipsCounts {
     timeout: "Number of hours before task timeout"
 
   }
+ 
   command <<<
+    reference=~{reference}
+ 
+    if [[ $reference == hg19 ]]; then 
+      bsGenome=BSgenome.Hsapiens.UCSC.hg19
+    else 
```

Comment:
I suggest making this an explicit check for $reference==hg38, should the workflow ever pick up data that is neither hg19 or hg38